### ISSUE TITLE
Render the semicolon usage in winter map

### ISSUE THREAD

**Issue by pbrideau** [waymarkedtrails/waymarked-trails-site#309]
I've been mapping summer and winter route as different relations
summer:
```
type=route
route=hiking
network=lwn
```
winter:
```
type=route
route=piste
piste:type=hike
piste:grooming=backcountry
```

But another user deleted one of the relation thinking it was a duplication (both relation have the same members).  We came to the conclusion to map these route as only one relation:
```
type=route
route=hiking
network=lwn
piste:type=hike
piste:grooming=backcountry
```
But the Waymarked Trails website only render the summer trail, not the winter one...

I cannot find the clear answer in the wiki about the way I should proceed, but the `piste:type` key should be allowed on any relation

[wiki on route=hiking](https://wiki.openstreetmap.org/wiki/Tag:route%3Dhiking)
[wiki on route=piste](https://wiki.openstreetmap.org/wiki/Tag:route%3Dpiste)
[wiki on piste:type](https://wiki.openstreetmap.org/wiki/Key:piste:type)

Here and example of the Waymarked Trails renderer:
[in summer](https://hiking.waymarkedtrails.org/#?map=15!45.4697!-71.1997)
[in winter](https://slopes.waymarkedtrails.org/#?map=15!45.4697!-71.1997)

Tested on this relation:
https://www.openstreetmap.org/relation/9263213

**Comment by lonvia** [comment]
The winter route requires a `route=piste` to be rendered. So you'd need something like `route=hiking;piste` to make the relation a multi-purpose route. For the hiking map that already works. For the winter map I could be persuaded to add support as well. Note however that other renderers might then render neither route because they don't understand the semicolon. The semicolon notation is controversial.

**Comment by pbrideau** [comment]
Thanks for pointing out the semicolon, I've used it in OSM as you suggested.  I'll rename the issue to be a reference about the semicolon for tracking purpose.  It would be great to have this supported on winter trails.  Also, I changed some other dupplications in my own mapping with ski and snowshoe route (as well as summer hiking route), also with the semicolon that would be nice to see supported if it is not a stretch to suggest:
```
route=hiking;piste
piste:type=hike;nordic
piste:grooming=backcountry
```
https://www.openstreetmap.org/relation/9255872#map=13/47.2011/-71.3904

As expected, this also does not work currently in WaymarkedTrails slopes:
https://slopes.waymarkedtrails.org/#?map=16!47.1773!-71.3749
